app/workers/publishing_api_gone_worker.rb

```diff
@@ -5,6 +5,8 @@ def perform(content_id, alternative_path, explanation, locale, allow_draft = fal
         .new.govspeak_to_html(explanation)
     end
 
+    alternative_path.squish! if alternative_path
```

Comment:
It probably won't cause any issues in this instance but using the `squish!` here is going to mutate the argument that has been passed in outside of the scope of this class. I think it would be 'better form' to not do that.